petab/lint.py

```diff
@@ -434,61 +499,35 @@ def measurement_table_has_observable_parameter_numeric_overrides(
     return False
 
 
-def assert_noise_distributions_valid(measurement_df: pd.DataFrame) -> None:
+def assert_noise_distributions_valid(observable_df: pd.DataFrame) -> None:
     """
-    Check whether there are not multiple noise distributions for an
-    observable, and that the names are correct.
+    Ensure that noise distributions and observable transformations
```

Comment:
observable transformations for observables :)

Reply:
Maybe this way the link to the `observableTransformation` column is more clearer... :)  I see your point.